src/components/client/campaigns/CampaignCard/CampaignCard.tsx

```diff
@@ -92,14 +92,15 @@ export default function ActiveCampaignCard({ campaign, index }: Props) {
         </StyledContent>
       </Link>
       <StyledCardActions disableSpacing>
-        {(campaignState === CampaignState.complete && !allowDonationOnComplete) || (
-          <DonateButton
-            href={routes.campaigns.oneTimeDonation(slug)}
-            variant="contained"
-            color="secondary">
-            {t('cta.support')}
-          </DonateButton>
-        )}
+        {campaignState === CampaignState.active ||
```

Comment:
May be I don't understand...
Isn't this going to hide the donation button for ACTIVE campaigns?

campaignState === CampaignState.active - this will be TRUE so the rest won't be evaluated?

Reply:
I agree